code/src/index.css

```diff
@@ -1,13 +1,508 @@
+}
+
+h1 {
+  font-family: 'Oswald', 'Montserat';
+  font-size: 2.1rem;
+  margin-top: 5px;
+  background-color: yellow;
+  transform: rotate(-10deg) translateX(10px);
+  display: block;
+  width: 280px;
+  margin-left: 30px;
+}
+
+.task-number-text {
+  padding-top: 1px;
+  margin-bottom: 5px;
+}
+
+.completed {
+  margin-top: 5px;
+  margin-bottom: 5px;
+  font-size: 0.9rem;
+}
+
+.moment-p {
+  margin-top: 5px;
+}
+
+.background-gif {
+  max-width: 100%;
+}
+
+.footer-p {
+  position: relative;
+  margin-top: 210px;
+  font-size: 0.7rem;
+}
+
+.week-day {
+  font-family: 'Oswald';
+  font-size: 1.2rem;
+  margin-top: 0;
+  margin-bottom: 0;
+  color: #444;
+}
+
+.no-todos-text {
+  color: rgb(92, 101, 117);
+  margin-top: 0;
+  font-size: 1rem;
+  padding-top: 15px;
+  padding-left: 30px;
+  padding-right: 30px;
+  font-family: montserrat;
+}
+
+.add-todo {
+  padding-top: 50px;
+  height: 25%;
+  background-color: rgb(235, 243, 249);
+  margin-top: 0;
+  display: flex;
+  flex-direction: column;
+}
+
+.task-number {
+  color: rgb(5, 157, 255);
+  font-weight: 700;
+  font-size: 1.2rem;
+}
+
+.todo {
+  position: relative;
+  height: 100vh;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background-color: rgb(235, 243, 249);
+}
+
+.left-todo {
+  position: relative;
+  flex-direction: column;
+  justify-content: left;
+  min-width: 80%;
+}
+
+.created-at {
+  position: absolute;
+  bottom: 10px;
+  left: 10px;
+  color: rgb(235, 243, 249);
+}
+
+.created-at-day {
+  position: absolute;
+  bottom: 30px;
+  left: 10px;
+  color: rgb(235, 243, 249);
+}
+
+.seperator {
+  border: solid 1px rgb(235, 243, 249);
+  transform: rotate(180deg) translateX(10px);
+}
+
+.right-todo {
+  flex-direction: column;
+  align-items: right;
+  justify-content: right;
+  width: 20%;
+  height: 100%;
+  border-bottom: 1px solid rgb(235, 243, 249);
+}
+
+.done {
+  position: relative;
+  display: flex;
+  width: 100%;
+  height: 50%;
+  height: 150px;
+  background-color: rgb(32, 178, 178);
+  color: #444;
+  border-bottom: 1px solid rgb(235, 243, 249);
+}
+
+.not-done {
+  position: relative;
+  display: flex;
+  width: 100%;
+  height: 50%;
+  height: 150px;
+  background-color: rgba(5, 157, 255, 0.6);
+  color: rgb(235, 243, 249);
+  border-bottom: 1px solid rgb(235, 243, 249);
+}
+
+.todo-text {
+  font-size: 1.1rem;
+  color: #444;
+  font-weight: 300;
+  font-family: 'Oswald';
+  position: absolute;
+  top: 10px;
+  left: 15px;
+  margin: 10px, 300px, 0 0;
+}
+
+.todo-section {
+  display: flex;
+  align-items: center;
+  align-self: center;
+  position: absolute;
+}
+
+.no-todos {
+  font-family: montserrat;
+  font-size: 1.5rem;
+  height: 100vh;
+}
+
+.todo-input {
+  margin-top: 20px;
+  font-family: montserrat;
+  font-weight: 500;
+  height: 48px;
+  width: 65%;
+  border-radius: 24px 0 0 24px;
+  border: none;
+  background-color: #444;
+  padding: 0 1rem 0 1rem;
+  color: #dedede;
+  outline: none;
+}
+
+.add-todo-button {
```

Comment:
<img width="451" alt="Screenshot 2021-11-25 at 22 39 11" src="https://user-images.githubusercontent.com/83988618/143501944-5e4da2f7-f3d5-4e3c-b2ab-4dfcd2f0c2ec.png">
For me the add-todo-button is not align with the input field in desktop view.